doc/introduction/operations.rst

```diff
@@ -473,3 +473,34 @@ Qutrit Observables
 
 :html:`</div>`
 
+.. _intro_ref_ops_pulse:
+
+Pulse-level operators
+---------------------
+
+If you would like to learn more about the implementation of pulse-level control in PennyLane, see the
+:mod:`~.pulse` module documentation.
+
+
+Pulse operator
+^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.pulse.ParametrizedEvolution
+
+:html:`</div>`
+
+This operator can be created via the :class:`~.ParametrizedHamiltonian` to Operator function:
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.evolve
+
+:html:`</div>`
```

Comment:
I'm not entirely sure if this is necessary. The first rendered description of `ParametrizedEvolution` has enough information in my opinion and also provides a reference to `evolve`, which is already included in the `Operator functions` section of the doc, and this is somewhat inconsistent with the structure of the rest of the doc.